app/assets/javascripts/trim_link.js

```diff
@@ -3,9 +3,47 @@
 
 var trimLink = {};
 
+
 // Prepare the callback on trim upload controls
 trimLink.trimFileUpload = function() {
   'use strict';
+  var $fileField = $('input[type=file]');
+
+  if ($('#pq-details').length) {
+    $('#status').hide();
+    $('#choices').on('click', function () {
```

Comment:
$('#status') and $('#choices') should be put in a variable, as they're used more than once